marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -572,29 +578,18 @@ public void handleProductArtifact(String version, String productId,
         repoUrl, mavenArtifact.getGroupId(), StringUtils.defaultIfBlank(snapshotVersionValue, version));
 
     if (StringUtils.isBlank(url)) {
```

Comment:
please remove this if condition. If some fields are null it'll get default value, this blank case will not happen